<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RePkgUiToolClass</class>
 <widget class="QMainWindow" name="RePkgUiToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>335</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>RePkgUiTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>搜索目录：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditSearchFolder"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>输出目录：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditOutputFolder"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>选项</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBoxCreateFolder">
         <property name="text">
          <string>为每个壁纸创建文件夹</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxSkipExistFile">
         <property name="text">
          <string>如果输出目录中已存在从对应壁纸提取出的图片文件，则跳过该pkg文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonAbout">
        <property name="text">
         <string>关于...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="startExtrackBtn">
        <property name="text">
         <string>开始提取</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonStop">
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBrowseSearchFolder">
   <property name="icon">
    <iconset resource="RePkgUiTool.qrc">
     <normaloff>:/RePkgUiTool/res/folder.ico</normaloff>:/RePkgUiTool/res/folder.ico</iconset>
   </property>
   <property name="text">
    <string>浏览...</string>
   </property>
  </action>
  <action name="actionBrowseOutputFolder">
   <property name="icon">
    <iconset resource="RePkgUiTool.qrc">
     <normaloff>:/RePkgUiTool/res/folder.ico</normaloff>:/RePkgUiTool/res/folder.ico</iconset>
   </property>
   <property name="text">
    <string>浏览...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RePkgUiTool.qrc"/>
 </resources>
 <connections/>
</ui>
